tests/unit/utilities/test_browser_handler.py

```diff
@@ -266,6 +308,33 @@ def test_open_chrome_non_windows(mock_set_browser_size, mock_add_argument, mock_
     check_mocked_functions_called(mock_chrome, mock_set_browser_size)
 
 
+@mock.patch("os.name", "nt")
+@mock.patch("selenium.webdriver.Firefox", side_effect=lambda **kwargs: "mock_firefox")
+@mock.patch("selenium.webdriver.FirefoxOptions")
+@mock.patch("uitestcore.utilities.browser_handler.BrowserHandler.set_browser_size")
+def test_open_firefox_windows(mock_set_browser_size, mock_firefoxoptions, mock_firefox):
```

Comment:
can you make it more obvious it's about Windows OS, not browser windows like tabs